app/models/imap_sync_log.rb

```diff
@@ -9,20 +9,26 @@ def self.levels
     @levels ||= Enum.new(:debug, :info, :warn, :error)
   end
 
-  def self.log(message, level, group_id = nil)
+  def self.log(message, level, group_id = nil, db = true)
```

Comment:
With arity of 4 I would consider named parameters. Don't have a strong opinion here so completely up to you

Reply:
Fair call but this method is kind of like a more custom/internal method anyway so will leave as is